utilities/common.py

```diff
@@ -98,8 +98,18 @@ def common_parser():
     parser.add_argument("-o", "--outfolder", type=str, default="", help="Output folder")
     parser.add_argument("--no-color-logger", dest="noColorLogger", action="store_true", default=False, help="Do not use logging with colors")
     parser.add_argument("-v", "--verbose", type=int, default=3, choices=[0,1,2,3,4],
-                        help="Set verbosity level with logging, the larger the more verbose (currently only for setup_test_logger enabled with --set-custom-logger)");
-    
+                        help="Set verbosity level with logging, the larger the more verbose (currently only for setup_test_logger enabled with --set-custom-logger)")
+    parser.add_argument("-e", "--era", type=str, choices=["2016PreVFP","2016PostVFP"], help="Data set to process", default="2016PostVFP")
+    parser.add_argument("--muonCorr", type=str, default="massfitData_lblidealMC", 
+        choices=["none", "trackfit_only", "trackfit_only_mctruth", "lbl", "massfit", "massfit_lbl", "massfitData_mctruth"], 
+        help="Type of correction to apply to the muons")
+    parser.add_argument("--muScaleMag", type=float, default=1e-4, help="Magnitude of dummy muon scale uncertainty")
```

Comment:
So these aren't actually valid options for the gen and low pileup scripts. Can you make a new function that is W/Z specific? So we have common_parser and wz_common_parser for example, where the latter calls the former which is a subset